doc/user/content/manage/access-control/_index.md

```diff
@@ -10,116 +10,298 @@ menu:
+Materialize console using the same authentication scheme and credentials across
+all systems in your organization.
 
-Authorization is determined in RBAC by your organization administrator. When you
-invite users to your Materialize organization, you have the option to give the
-user more elevated privileges in the Materialize administrator console.
+To configure SSO for your Materialize organization, follow [this step-by-step
+guide](/manage/access-control/sso).
 
-## RBAC structure
+[//]: # "NOTE(morsapaes) This sits kind of awkward in here, but feels like the
+best place to plug it. Need to add some more meat if we keep it."
 
-RBAC in practice is a group of roles with assigned privileges.
-You can assign specific users to roles or assign privileges to users to inherit
-from other roles.
+### Using an external secret store
 
-### Roles
+Although Materialize does not integrate directly with external secret stores,
+it’s possible to manage this integration via [Terraform](/manage/terraform).
 
-A role is a collection of privileges you can apply to users. Roles make it
-easier to assign or revoke privileges on Materialize objects. You can group
-users into specified roles with different levels of privileges and adjust those
-privileges to ensure they have the correct level of access to objects.
+Check the [Terraform documentation](/manage/terraform/#external-secret-stores)
+for more details on how to integrate with common external secret stores, like
+HashiCorp Vault or AWS Secrets Manager.
 
-### Role attributes
+## Role-based access control (RBAC)
 
-Role attributes are actions available to any role you create. Attributes are
-independent of any other object in Materialize and apply to the entire
-organization. You can edit these actions when you create the role:
+[//]: # "NOTE(morsapaes) These instructions assume that RBAC is enabled at the
+time of region creation. For existing regions, we assume that migration to RBAC
+is whitegloved."
 
-| Name              | Description                                                                 |
-|-------------------|-----------------------------------------------------------------------------|
-| `INHERIT`         | **Read-only.** Can inherit privileges of other roles.                       |
+The default level of access to the database is determined by the
+organization-level role a user is assigned on invitation (`Organization Admin` or `Organization Member`).
 
-PostgreSQL uses role attributes to determine if a role is allowed to execute certain statements. In
-Materialize these have all been replaced by system privileges.
+The first user in an organization, and subsequently any user that is assigned
+`Organization Admin`, is a database _superuser_, and has unrestricted access to
+all resources in a Materialize region. Users that are assigned `Organization
+Member` are restricted to a default set of basic privileges that need to be
```

Comment:
Where do we list what these default basic privileges are?

Reply:
That's a good point! I'll add a link to the section below.